astrosat_users/views/__init__.py

```diff
@@ -13,6 +13,7 @@
     CustomerDetailView,
     CustomerUserListView,
     CustomerUserDetailView,
+    CustomerUserInviteView,
```

Comment:
I keep forgetting the benefit of importing in this file. Is it so you don't have to import these again in any other file, within this directory? But instead of importing everything, only stuff that is used in multiple files, but not necessarily all? I get that code in here is implicitly run and for code that does calculations, retrieving of data etc, it makes sense. I just can't remember the benefits of doing imports here.

Reply:
There are various reasons.  I usually do it b/c it's a logical grouping.  So if I type

```
from astrosat_users.views import *
```

I know that I will get all the view classes but none of the internal helper classes I don't care about. 

For Django, I just have a habit of organising my modules like this.  So I can type `from app.serializers import MySerializer` anywhere in the code w/out having to worry about the specific module that `MySerializer` is defined in.

Reply:
ah yes, you have explained that bit before and I had forgotten it. So you effectively are re-exporting a class from a sub-module e.g. view, serializer etc in the `__init__.py` file, so it can be imported from here, instead of the real file it is defined in. This is much like what we can do (although I'm not a big fan) in JavaScript. If you have an `index.js` file in a directory, in there, you can re-export, for instance in `orbis/map/index.js` does just this.